cfps/CFP-002-template.md

```diff
@@ -0,0 +1,84 @@
+# CFP-002: Template
+
+**SIG: SIG-NAME**
+
+**Begin Design Discussion:** YYYY-MM-DD
+
+**End Design Discussion:** YYYY-MM-DD
```

Comment:
I think the idea behind these dates is to bound the time to evaluate the CFP, however in practice we have a lot of difficulty picking these dates and the evaluation depends heavily on the experts in the relevant areas. Given that this will be committed into git, how should a contributor pick the end date? I wonder if we should drop that one.

Reply:
I agree "time when the last change to the file was merged" seems like a more natural way to keep track of when the discussion finished.

Reply:
I think that makes sense. I'm going to drop this since the last change can easily be found from the last commit in Git